src/bridge-did/src/operations/btc.rs

```diff
@@ -22,13 +22,17 @@ pub enum BtcBridgeOp {
         eth_address: H160,
         amount: u64,
     },
+    SignMintOrder {
+        eth_address: H160,
+        order: MintOrder,
```

Comment:
As far as I understand, the `eth_address` is equal to `order.recipient`. So, we can remove it.